### PR TITLE
fix: Make the Superset MySQL database utf8mb4

### LINKED ISSUE #452: Initialize the `superset` database to utf8mb4
MySQL is initialized to `latin1`.

This prevents us from adding Arabic or other Unicode.'

On the other hand `edxapp` is being created correctly:

```
CREATE DATABASE `edxapp` /*!40100 DEFAULT CHARACTER SET utf8mb4 */‏‏
```

@Ian2012 would you mind taking a look?

## Context

We're aiming to add full Arabic support for both Aspect and superset for @nelc:

   * RTL: https://github.com/apache/superset/issues/25258
   * Unicode support (this issue)
   * Translations of the Aspect dashboard UI (Needs contribution)
   * Translations for the Superset UI: https://github.com/apache/superset/issues/25258

### PR DESCRIPTION
**Pull request by bmtcril**
Fixes #452

### CODE REVIEW

File: tutoraspects/templates/aspects/jobs/init/init-mysql.sh [hunk 1]
```diff
@@ -14,8 +14,8 @@ do
 done
 echo "MySQL is up and running"
 
-# edx-platform database
-mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }};"
+# Superset MySQL database, by default points to the same host as edx-platform
+mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} /*!40100 DEFAULT CHARACTER SET utf8mb4 */;"
```
Comment (Ian2012) [inline tutoraspects/templates/aspects/jobs/init/init-mysql.sh:18]: The comment will have no effect over the charset:

```suggestion
mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci;;"
```

**Review by Ian2012 — CHANGES_REQUESTED** [review]

File: tutoraspects/templates/aspects/jobs/init/init-mysql.sh (hunk 1 above)
Comment (bmtcril) [reply tutoraspects/templates/aspects/jobs/init/init-mysql.sh:18]: @Ian2012 the comment is actually a MySQL directive that tells it to execute the command only on versions >= 4.1.0 (where these charsets were introduced, I believe). It should work correctly on any modern MySQL. `utf8mb4_0900_ai_ci` is the default collation for `utf8mb4` so I don't think adding that will change anything.

**Review by Ian2012 — APPROVED** [review]

**Review by omar-nelc — COMMENTED** [review]
Thanks @Ian2012 and @bmtcril!!

Yes, to confirm the `CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} /*!40100 DEFAULT CHARACTER SET utf8mb4 */;` should work. 

As well as `CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} DEFAULT CHARACTER SET utf8mb4;`

This PR should fix the issues we have :)

**Comment by OmarIthawi** [comment]
Thanks @Ian2012 and @bmtcril!!

Yes, to confirm the `CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} /*!40100 DEFAULT CHARACTER SET utf8mb4 */;` should work. 

As well as `CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} DEFAULT CHARACTER SET utf8mb4;`

This PR should fix the issues we have :)

**Pull request merged**